### PR TITLE
Fix norentrancy and minor refactoring

### PR DESCRIPTION
**Pull request by ali2251**

### CODE REVIEW

File: contracts/VoucherKernel.sol [hunk 1]
```diff
@@ -59,7 +59,7 @@ contract VoucherKernel is IVoucherKernel, Ownable, Pausable, UsingHelpers {
     mapping(address => uint256) public tokenNonces; //mapping between seller address and its own nonces. Every time seller creates supply ID it gets incremented. Used to avoid duplicate ID's
     mapping(uint256 => VoucherPaymentMethod) public paymentDetails; // tokenSupplyId to VoucherPaymentMethod
 
-    bytes32[] public promiseKeys;
```
Comment (HristiyanG) [inline contracts/VoucherKernel.sol:62]: Why should we conceal the promiseKeys?

**Comment by HristiyanG** [comment]
In general looking good. Just something to clarify - this PR states it fixes reentrancy, but there are a lot of linting changes. Can you make sure you run the pipeline locally before a PR, so we can all have similar linting?

**Review by hswopeams — CHANGES_REQUESTED** [review]
Make promiseKeys mapping public again.

File: contracts/VoucherKernel.sol [hunk 2]
```diff
@@ -59,7 +59,7 @@ contract VoucherKernel is IVoucherKernel, Ownable, Pausable, UsingHelpers {
...
+    bytes32[] promiseKeys;
```
[the remark this answers is not in the document]
Comment (ali2251) [reply contracts/VoucherKernel.sol:62]: Sure, will add public back

**Comment by ali2251** [comment]
I am not sure why the linting changes occured and why they arent being fixed by formatter and pipeline is passing

**Comment by HristiyanG** [comment]
Is this still outstanding? Shall we merge / close this one? @thecryptofruit, @hswopeams

**Review by hswopeams — APPROVED** [review]

**Pull request merged**